src/main/kotlin/uk/gov/justice/probation/courtcaseservice/service/CaseWorkflowService.kt

```diff
@@ -24,6 +24,18 @@ class CaseWorkflowService(val hearingRepository: HearingRepository, val courtRep
             })
     }
 
+    fun assignHearingOutcomeTo(hearingId: String, assignedTo: String, assignedToUuid: String) {
+        hearingRepository.findFirstByHearingId(hearingId).ifPresentOrElse(
+                { hearingEntity: HearingEntity ->
+                    hearingEntity.hearingOutcome.assignTo(assignedTo, assignedToUuid)
+                    hearingEntity.hearingOutcome.updateState(HearingOutcomeItemState.IN_PROGRESS.name)
```

Comment:
Can this not be moved into of assignTo? this state is implied in the assign flow?

Reply:
I would rather rename this function name to something like assignAndUpdateOutcomeToInProgress().

Reply:
Updated - Thanks guys!

Reply:
in progress is a state not outcome and the state transition is implied for assign flow and is a behind the scene presentation of the outcome which doesn't need to reflect in the method. 
Approving though